packages/monorepo-scripts/src/prepublish_checks.ts

```diff
@@ -17,6 +17,20 @@ async function prepublishChecksAsync(): Promise<void> {
     await checkChangelogFormatAsync(updatedPublicPackages);
     await checkGitTagsForNextVersionAndDeleteIfExistAsync(updatedPublicPackages);
     await checkPublishRequiredSetupAsync();
+    await checkDockerHubSetupAsync();
+}
+
+async function checkDockerHubSetupAsync(): Promise<void> {
+    try {
+        utils.log('Checking that the user is logged in to docker command...');
+        await execAsync(`echo "$DOCKER_PASS" | docker login -u $DOCKER_USERNAME --password-stdin`);
```

Comment:
[This resource](https://gist.github.com/kizbitz/175be06d0fbbb39bc9bfa6c0cb0d4721#file-dockerhub-v2-api-organization-sh-L17) could help us check if the user also has permission to write to the repository, which could be forgotten.

Successful `REPO_LIST` would contain the image under the Org.

Assuming ORG=0xorg:
```
curl -s -H "Authorization: JWT ${TOKEN}" https://hub.docker.com/v2/repositories/${ORG}/?page_size=100 | jq -r '.results|.[]|.name'
verdaccio
devnet
ganache-cli
launch-kit-ci
order-watcher
```